src/test/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -167,7 +167,7 @@ public String toString() {
     public static File createTempFile(final String name, final String extension) {
         try {
             final File file = File.createTempFile(name, extension);
-            file.deleteOnExit();
+            //file.deleteOnExit();
```

Comment:
Restore this line -- temporary files should be deleted on exit, and callers of this method rely on that behavior.